### PR TITLE
Railroad fix

### PR DESCRIPTION
**Pull request by nopunin10did**
## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Noticed that I'd forgotten to rename a couple files after moving them to a rev0 folder.

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/nopunin10did/railroad/rev0/rules.mk [hunk 1]
```diff
@@ -20,3 +20,5 @@ BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
 BLUETOOTH_ENABLE = no       # Enable Bluetooth
 AUDIO_ENABLE = no           # Audio output
+
+LAYOUTS = prototype
```
Comment (fauxpark) [inline keyboards/nopunin10did/railroad/rev0/rules.mk:24]:
```suggestion
```

File: keyboards/nopunin10did/railroad/rev0/rev0.h [hunk 2]
```diff
@@ -50,4 +50,4 @@
    { K0900, K0901, K0902, K0903, K0904, K0905, K0906, K0907, K0908, K0909, KC_NO }  \
 }
 
-
+#define LAYOUT_prototype LAYOUT
```
Comment (fauxpark) [inline keyboards/nopunin10did/railroad/rev0/rev0.h:53]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

File: keyboards/nopunin10did/railroad/rev0/readme.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# The Railroad
+
+![The Railroad](https://i.imgur.com/B8OjXmyl.jpg)
+
+A long keyboard with ortholinear alphas and semi-standard modifier keys. For more information, join the [NoPunIn10Key Discord](https://discord.gg/sku2Y6w).
+
+* Keyboard Maintainer: [Alex Ronke](diplomacyvariants.wordpress.com), also known as NoPunIn10Did.
```
Comment (noroadsleft) [inline keyboards/nopunin10did/railroad/rev0/readme.md:7]: Fixes the first link, and also links to your GitHub account.

```suggestion
* Keyboard Maintainer: [Alex Ronke](https://diplomacyvariants.wordpress.com/), also known as [NoPunIn10Did](https://github.com/nopunin10did). 
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Comment by nopunin10did** [comment]
@noroadsleft Change committed.

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**